identifier origin
bool[] bools
uint8[] uint8s
int8[] int8s
uint16[] uint16s
int16[] int16s
uint32[] uint32s
int32[] int32s
uint64[] uint64s
int64[] int64s
float32[] float32s
float64[] float64s
string[] strings
